keyboards/cest73/tkm/rules.mk

```diff
@@ -0,0 +1,30 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = halfkay
+
+# Build Options
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE = yes  # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no    # Mouse keys
+EXTRAKEY_ENABLE = yes   # Audio control and System control
+CONSOLE_ENABLE = no     # Console for debug
+COMMAND_ENABLE = no     # Commands for debug and configuration
+SLEEP_LED_ENABLE = no   # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes       # USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = yes  # Enable keyboard backlight functionality
+UNICODE_ENABLE = yes    #
+UNICODEMAP_ENABLE = no  # send Unicode characters using X(<map index>) in your keymap. All code points (up to 0x10FFFF) are supported
+UCIS_ENABLE = no        # send Unicode characters by inputting a mnemonic. All possible code points (up to 0x10FFFF) are supported
```

Comment:
Not sure whether the Unicode support is supposed to be enabled at the keyboard level (at the moment there are 62 keyboards in the repo that have `UNICODE_ENABLE = yes`, and technically this option, unlike the two other Unicode support options, could work in the configurator if you use the Any key to enter the codes). However, note that the `UNICODE_MODE_FORWARD` and `UNICODE_MODE_REVERSE` keycodes won't work without defining `UNICODE_SELECTED_MODES`; the keycodes which select the mode directly (like `UNICODE_MODE_LNX`) should work, however.

Reply:
`UNICODE_ENABLE` is fine at keyboard level, although it may not be terribly useful as neither Configurator nor VIA currently support `UC()` keycodes. The other two cannot be enabled at keyboard level because they require custom code in the keymap (`unicode_map[]`/`ucis_symbol_table[]`) and trying to build the keyboard in the Configurator will fail.